site/Gemfile.lock

```diff
@@ -48,22 +48,20 @@ GEM
       jekyll (>= 3.3, < 5.0)
     jekyll-watch (2.2.1)
       listen (~> 3.0)
-    kramdown (2.3.0)
-      rexml
+    kramdown (1.17.0)
```

Comment:
Kramdown version went backward?

Reply:
There was a PR that updated because of a CVE - https://github.com/vmware-tanzu/sonobuoy/pull/1154

Reply:
hah -- I see what's going on. Kramdown was referenced indirectly through Jekyll and [Jekyll v3.8.7 uses Kramdown `~> 1.14`](https://github.com/jekyll/jekyll/blob/v3.8.7/jekyll.gemspec#L39). Bundle autoresolves Kramdown back to 1.17 because it was indirectly references.

Looks like the path forward is to actually upgrade Jekyll so that Kramdown goes up without version conflict.

Reply:
Turns out we've been using [`redcarpet` as Markdown processor](https://github.com/vmware-tanzu/sonobuoy/blob/185c859371c7dfa6858906ab6e1f220c51cfaf0b/site/_config.yml#L13) while [its support has been dropped by Jekyll as of v4](https://github.com/jekyll/jekyll/issues/6981).

Going to switch it to Kramdown and see how if anything breaks.